extensions/banner/deployment/src/main/java/io/quarkus/banner/deployment/BannerConfig.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.banner.deployment;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "banner", phase = ConfigPhase.BUILD_TIME)
+public class BannerConfig {
+
+    /**
+     * The path of the default quarkus banner
+     */
+    public static final String DEFAULT_BANNER_PATH = "default_banner.txt";
```

Comment:
Can we utilize the `defaultValue` from `@ConfigItem` for this?

Reply:
No. Otherwise it will ask for default_path parameter from user. Besides Its default for Quarkus, so IG it makes more sense to be static final

Reply:
I still don't understand why it needs to be a separate field instead of just using: https://github.com/quarkusio/quarkus/blob/master/core/runtime/src/main/java/io/quarkus/runtime/annotations/ConfigItem.java#L55.
Something like

```
@ConfigItem(defaultValue = "default_banner.txt")
public String path;
```

Why would that not work?

Reply:
Default value is present in Quarkus-banner resources for that only classLoader works
but for the user provided banner. It exists in their project resources for that, we need `contexrClassLoader`. If I give default value here, there won't be no way to differentiate between the two

Reply:
OK, thanks

Reply:
Sorry my bad. Your suggestion is working. I used it default value directly as string and didn't use annotation's default value. 
Thanks for telling me. improving the code :sweat_smile:

Reply:
Great to hear :wink: